.github/workflow/linters.yml

```diff
@@ -0,0 +1,48 @@
+name: Linters
```

Comment:
- [ ] Actually, the folder name for the linter set is incorrect - your folder structure is .github / workflow / `linters.yml`, but it should be workflows, with `s` in the end. Please, fix it and then you will see Actions (exactly, that Github Actions, which was the previous reviewer mentioned). Here is what it will look like in the end when linter will check and will not find any errors (but remember, it should pass without errors - if the error will available, please, fix it before the next submission)

> This is how linter looks like after successful checking (Actions)
<img width="927" alt="Screen Shot 2022-02-09 at 00 48 16" src="https://user-images.githubusercontent.com/49064106/153073110-b861c30e-5bca-462f-b3ed-22e6279ac150.png">